app/routes/realtime_audio.py

```
@@ -0,0 +1,98 @@
+
+@router.websocket("/ws/audio")
+async def websocket_audio_stream(websocket: WebSocket):
+    await websocket.accept()
+    buffer = AudioStreamBuffer()
+    transcriber = TranscriberService()
+
+    async def try_transcribe():
+        """ Transcribe and send result back if buffer has content """
+        if not buffer.is_empty():
+        # Decode and write raw audio to a temp file
+            audio_bytes = b''.join([base64.b64decode(c) for c in buffer.get_and_clear()])
```

Ruff (0.11.9)

Undefined name `base64`

(F821)